src/config-site/scratlantis.cmake

```diff
@@ -135,6 +112,18 @@ VISIT_OPTION_DEFAULT(VISIT_HDF5_MPI_LIBDEP ${VISITHOME}/szip/2.1/${VISITARCH}/li
 ##
 SETUP_APP_VERSION(ADIOS 1.13.1)
 VISIT_OPTION_DEFAULT(VISIT_ADIOS_DIR ${VISITHOME}/adios/${ADIOS_VERSION}/${VISITARCH})
+##
+## BLOSC2
+##
+VISIT_OPTION_DEFAULT(VISIT_BLOSC2_DIR ${VISITHOME}/blosc2/2.11.3/${VISITARCH})
```

Comment:
How did you generate this cmake file? I thought I fixed the newline issue for blosc2. There should be a newline before line 115. Not that it matters.

Reply:
its from my last release build for intel-mac for 3.4.2

Reply:
Looks like I changed this on December 4th:
![image](https://github.com/user-attachments/assets/2c8f0d5b-b3c6-4881-bf76-c17c5206afbc)

So I don't understand how you are missing the newline, unless you are used a version of the RC from before then.

Reply:
Well, that file was generated a few weeks ago so I am assuming it predates your change to address it.